prow/cmd/deck/main.go

```diff
@@ -36,12 +36,14 @@ import (
 	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/pjutil"
+	"k8s.io/test-infra/prow/pluginhelp"
 )
 
 var (
 	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
 	buildCluster = flag.String("build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
 	tideURL      = flag.String("tide-url", "", "Path to tide. If empty, do not serve tide data.")
+	hookURL      = flag.String("hook-url", "http://hook:8888/plugin-help", "Path to hook plugin help endpoint.")
```

Comment:
Do not hardcode this here and make the behavior opt-in, similarly to what was done for `tide-url`, otherwise you are tying the deployment of both components together.

Reply:
Fixed.